scripts/zkai_attention_kv_risc0_semantics_receipt_gate.py

```
@@ -0,0 +1,789 @@
+        "--quiet",
+        "--manifest-path",
+        str(PROGRAM_MANIFEST),
+        "-p",
+        "host",
+        "--",
+        mode,
+        str(input_path),
+        str(receipt_path),
+        str(summary_path),
+    ]
+    result = subprocess.run(command, check=False, text=True, capture_output=True, timeout=900, cwd=ROOT, env=env)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)